ClassPaulita/GuessTheCardGame.cs

```diff
@@ -26,39 +160,60 @@ private bool UserChoseToQuit(char userChoice)
             return userChoice == 'q' || userChoice == 'Q';
         }
 
-        private bool IsUserChoiceValid(char userChoice)
-        {                      
+        private char GetUserGuess(Card previousCard)
+        {
+            Console.Write($"\n\n\tIs the next card greater than {previousCard}(y or n)? ");
+            char userGuess = Convert.ToChar(Console.ReadLine());
```

Comment:
This will fail when the user types more than one character...
![image](https://user-images.githubusercontent.com/43797367/61090978-728f4480-a40e-11e9-8d6d-42707d45e09c.png)

Reply:
I was able to fix this issue. Thank you!